src/app/common/data_processing.py

```diff
@@ -34,3 +36,27 @@ def format_ncbi_summary() -> None:
         "Status",
     ]
     st.session_state[NCBI_DF] = st.session_state[NCBI_DF].reindex(columns=column_order)
+
+
+def get_top_organisms_counts() -> str:
+    """Get the top N organisms and their counts in the session's NCBI DataFrame.
+
+    Returns:
```

Comment:
Seems like you're back to the google docstring convention here??

Reply:
Thank you for catching this!